septima-js-sql-driver-postgresql/build.gradle

```diff
@@ -1,5 +1,3 @@
-version = '0.0.7'
-
 dependencies {
     compileOnly 'org.postgresql:postgresql:42.1.4'
```

Comment:
Why do use expired version? Current version is 42.2.5.

Reply:
Expired? Are you serious?